src/components/InjectedComponents/DecryptedPost.tsx

```diff
@@ -41,7 +46,7 @@ function DecryptPostSuccess({ data, people, ...props }: DecryptPostSuccessProps)
                 <>
                     {ShareMenu}
                     {geti18nString('decrypted_postbox_title')}
-                    <Box flex={1} />
+                    <Box flex={1}/>
```

Comment:
Your prettier breaks again?

Reply:
ok actually no prettier running during this change. prettier only run during an npm script running.